### PR TITLE
expose resource to view

### PR DESCRIPTION
**Pull request by Uysim**
global expose on create render 
Example
```erb
<% if @resource.present? %>
  <p>Message</p>
<% else %>
  <p>Otherg</p>
<% end %>
```

Related to: #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,15 @@ app/views/passwordless/sessions/create.html.erb
...
 ```
 
+Example on create.html.erb
+```erb
+<% if @resource.present? %>
+  <p>Message</p>
+<% else %>
+  <p>Other Message</p>
+<% end %>
+```
```
Comment (mikker) [inline README.md]: How about

```
If you'd like to let the user know whether or not a record was found, `@resource` is provided to the view. You may override `app/views/passwordless/session/create.html.erb` for example like so:

```erb
<% if @resource %>
  User found, check your inbox
<% else %>
  No user found with the provided email address?
<% end %>
```

Comment (Uysim) [reply README.md]: I changed it

Comment (mikker) [reply README.md]: I meant to include the first line too instead of “example create.erb”

Comment (Uysim) [reply README.md]: Sorry didn't notice. Not I changed it

**Comment by mikker** [comment]
Wonderful! Thanks for this.

**Pull request merged**